internal/parser/planparserv2/plan_parser_v2_test.go

```diff
@@ -204,25 +204,44 @@ func TestExpr_Like(t *testing.T) {
 	helper, err := typeutil.CreateSchemaHelper(schema)
 	assert.NoError(t, err)
 
-	exprStrs := []string{
-		`VarCharField like "prefix%"`,
-		`VarCharField like "equal"`,
-		`JSONField["A"] like "name*"`,
-		`$meta["A"] like "name*"`,
-	}
-	for _, exprStr := range exprStrs {
-		assertValidExpr(t, helper, exprStr)
-	}
+	expr := `A like "8\\_0%"`
+	plan, err := CreateSearchPlan(helper, expr, "FloatVectorField", &planpb.QueryInfo{
+		Topk:         0,
+		MetricType:   "",
+		SearchParams: "",
+		RoundDecimal: 0,
+	}, nil)
+	assert.NoError(t, err, expr)
+	assert.NotNil(t, plan)
+	fmt.Println(plan)
+	assert.Equal(t, planpb.OpType_PrefixMatch, plan.GetVectorAnns().GetPredicates().GetUnaryRangeExpr().GetOp())
+	assert.Equal(t, "8_0", plan.GetVectorAnns().GetPredicates().GetUnaryRangeExpr().GetValue().GetStringVal())
+
+	expr = `A like "8_\\_0%"`
```

Comment:
what about A like 8_\_0%"? 
I'm assuming 8_\_0% and 8_\\_0% is the same?

Also things like 8_\\\_0%

Reply:
the pattern has be changed by github,  I mean one Backslash and three Backslash

Reply:
`_` is a wildcard in Milvus. If you use `8__%`, it will match any string that starts with `8`. If it is `8_\\_%`, with the escape character, the second underscore is treated as a normal character, and it will only match strings with `8._` . If it is `8_\\\\_%`, the backslash will also be treated as a normal character, and it will only match strings with `8.\_`. ( `.` represents any character.)